hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -126,7 +126,7 @@
 
   public static final ConfigProperty<Boolean> POPULATE_META_FIELDS = ConfigProperty
       .key(METADATA_PREFIX + ".populate.meta.fields")
-      .defaultValue(true)
+      .defaultValue(false)
```

Comment:
Why is this changing?

Reply:
This is only for metadata table. Its intentional that we are disabling meta fields, that is enabling virtual keys for metadata table.